docs/api.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[api]]
+== Public API
+
+The Elastic APM Python agent has several public APIs.
+Most of these APIs are not needed when using one of our supported frameworks (Django and Flask),
+but they allow customized usage.
+
+[break]
+[[client-api]]
+=== Client API
+
+[break]
+[[client-api-init]]
+==== Instantiation
+
+To create a `Client` instance, import it and call its constructor:
+
+[source,python]
+----
+from elasticapm import Client
+
+client = Client({'APP_NAME': 'example'}, **defaults)
+----
+
+[break]
+===== Arguments
+
+ * `config`: A dictionary, with key/value configuration. For the possible configuration keys, see <<configuration, Configuration>>.
+ * `**defaults`: default values for configuration. These can be omitted in most cases, and take the least precedence.
+
+[[error-api]]
+=== Error API
+
+[break]
+[[client-api-capture-exception]]
+==== `Client.capture_exception()`
```

Comment:
This `Client.capture_exception()` looks like a class method, but then you actually call `client.capture_exception()`

Reply:
I used a capital C in all the titles that of methods on `Client`, I thought it would be clear that it means "method on the Client class", didn't really think of the issue that it could be mistaken as a class method.

Class methods (or static methods) are used relatively seldom in Python, so I don't think most readers would be confused.

Reply:
ok, maybe it is just me then, coming from a ruby background.